uiowa/uiowa
# Admissions - AoS Permissions Update

Resolves https://github.com/uiowa/uiowa/issues/3380
Resolves #3248

This removes the AoS Block from the AoS feature split because it failed on import due to several fields not existing in the feature split. I don't think any other sites besides Admissions are using this feature.

## To Test

- pull, sync the default site, login, enable the areas of study feature split, `cim` - the import should run successfully
- as a webmaster with the area of study feature split enabled, you should be able to add, publish and then unpublish an area of study without error.
- sync admissions, as a webmaster you should be able to unpublish an area of study without error.

## Note

~~I created this PR separately from https://github.com/uiowa/uiowa/pull/3379 because I thought that was blocked. It also has changes to Admissions permissions, so which ever gets merged first will have to be updated in the other.~~
